plugin.video.milbtv/addon.xml

```diff
@@ -19,24 +19,7 @@
         </summary>
         <description lang="en_GB">Watch live and archived minor league baseball games with your MiLB.TV account</description>
         <disclaimer lang="en_GB">Requires a valid MiLB.TV account (separate from MLB.TV)</disclaimer>
-        <news>
-            - options to automatically skip breaks and non-action pitches
-            - option to start at specific inning or start live from beginning
-            - pad the end of archive videos if no spoilers (to avoid timeline spoilers)
-            - see scores at inning category
-            - hide bottom of 9th / extra innings as spoilers
-            - labels for suspended/resumed games
-            - game icons and fanart
-            - probable pitchers
-            - added probable pitchers to game list titles
-            - doubleheader/TBD start times
-            - additional AddonIsEnabled detection for inputstream adaptive
-            - perfect game / no hitter flags
-            - code cleanup
-            - removed failing, unnecessary epg api call
-            - option to disable video length padding (in case of proxy conflicts)
-            - added additional action play type in skip monitor
-        </news>
+        <broken>MiLB.TV has been discontinued. Those games are now available in MLB.TV instead.</broken>
```

Comment:
Please use the correct tag: https://kodi.wiki/view/Addon.xml#%3Clifecyclestate%3E